DNN Platform/Components/ClientDependency/Source/DependencySorter.cs

```diff
@@ -10,6 +10,30 @@ namespace ClientDependency.Core
     /// </summary>
     internal static class DependencySorter
     {
+
+        public static IList<IClientDependencyFile> FilterDependencies(IList<IClientDependencyFile> dependencies)
+        {
+            if (dependencies.Any(f => f.Name != ""))
+            {
+                var newList = dependencies.Where(f => f.Name == "").ToList();
+                var frameworks = dependencies.Where(f => f.Name != "").GroupBy(f => f.Name.ToLower());
+                foreach (var framework in frameworks)
+                {
+                    var topPriority = framework.FirstOrDefault(d => d.ForceVersion);
```

Comment:
What happens if there are multiple files marked as `ForceVersion`?  Should this sort by `ForceVersion` then by `Version`?